src/main/java/net/openhft/chronicle/core/io/VanillaReferenceCounted.java

```diff
@@ -118,12 +124,13 @@ public void release(ReferenceOwner id) throws ClosedIllegalStateException {
         }
     }
 
-    public void callOnRelease() throws ClosedIllegalStateException {
+    private void callOnRelease() throws ClosedIllegalStateException {
```

Comment:
This was only used by VanillaReferenceCounted, it seems like an internal method to me